### ISSUE TITLE
Admin Config: Implement a Groups UI Picker 

### ISSUE THREAD

**Issue by marianunez** [mattermost/mattermost-plugin-custom-attributes#28]
When configuring a custom attribute in the System Console, you must provide the group ids in a simple text box. Implement a picker that can allow the user to search by group name and select based on a filtered list (like it currently does with the users).

**Comment by hanzei** [comment]
@marianunez What do you think about making this a HW?

**Comment by hanzei** [comment]
@marianunez Does this require implementing an go API to fetch the groups?

**Comment by marianunez** [comment]
@hanzei No, there are already the required endpoints to get and search groups here: https://api.mattermost.com/#tag/groups%2Fpaths%2F~1groups%2Fget but may require adding it to mattermost-redux.
